README.md

```diff
@@ -0,0 +1,113 @@
+# CreateProcessW
+
+This crate provide an API similar to `std::process` to create and handle process
+using the [windows-rs][windows-rs] crate and the Windows API (see [this example][creating-processes]).
+
+# Usage
+
+Add the following to your `Cargo.toml`:
+
+```toml
+[dependencies]
+CreateProcessW = "0.1.0"
+```
+
+This crate doesn't follow Rust's naming convention, if you want to avoid the
+`non_snake_case` name, you can use the following:
+
+```toml
+[dependencies]
+create_process_w = { version = "0.1.0", package = "CreateProcessW" }
+```
```

Comment:
Only provide the solution with the rename!! XD nobody wants to deal with the non_snake_case